docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,120 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **If**
+
+    The if (tensor_if) element allows creating conditional branches based on tensor values. For example, you may skip frames if there is no object detected with high confidence. The input and output stream data type is either `other/tensor` or `other/tensors`.
+
+    - **Properties**
+      - `compared-value`: Specifies the compared value and is represented as operand 1 from input tensors.
+        - A_VALUE: Decided based on a single scalar value.
+        - TENSOR_AVERAGE_VALUE: Decided based on an average value of a specific tensor.
+        - CUSTOM: Decided based on a user-defined callback.
+      - `compared-value-option`: Specifies an element of the nth tensor or you can pick one from the tensors.
+        - [C][W][H][B],n: Used for A_VALUE of the compared-value, for example 0:1:2:3,0 means [0][1][2][3] value of first tensor.
+        - nth tensor: Used for TENSOR_AVERAGE_VALUE of the compared-value, and specifies which tensor is used.
+      - `supplied-value`: Specifies the supplied value (SV) from the user.
+        - SV
+        - SV1, SV2: Used for RANGE operators.
```

Comment:
@gichan-jang Whats RANGE, Why this is in capital ? 

Cant we write it in small ? Or is it deliberately written as capital?

Can we rather use code styling for it ?

Reply:
How about change to `RANGE_*`?

Reply:
Yes if thats how you write it actually, It seems fine. 
```

Reply:
> ~How about change to `RANGE_*`?~
> 
> Um, I think this change looks better. PTAL.

I agree

Reply:
Updated.